app/models/domain_count.rb

```diff
@@ -0,0 +1,12 @@
+class DomainCount < ActiveRecord::Base
```

Comment:
Consistently indent two spaces per level, please.

Reply:
fixed this, some problem with my local text editor may be, Thank you !

Reply:
👍